src/Components/CreateImageWizard/CreateImageWizard.js

```diff
@@ -516,6 +516,10 @@ const requestToState = (composeRequest, distroInfo, isProd, enableOscap) => {
     if (enableOscap) {
       formState['oscap-profile'] =
         composeRequest?.customizations?.openscap?.profile_id;
+      formState['oscap-profile-name'] =
+        composeRequest?.customizations?.openscap?.profile_name;
+      formState['oscap-profile-description'] =
+        composeRequest?.customizations?.openscap?.profile_description;
```

Comment:
In this function we are loading a `composeRequest` into a state for the wizard. However these two values are not going to endup in the request, so they will never get to be loaded. It is thus safe to remove these lines.

But the fact that the data can't be loaded from a request will have side effects for the wizard.

It means that, upon recreating a request, when entering the `oscap` step, we will have the issue where the `profileName` is initialized to an empty string. 

Let's fix this :slightly_smiling_face: